lte/gateway/python/magma/smsd/relay.py

```diff
@@ -20,6 +20,7 @@
 from lte.protos.mconfig.mconfigs_pb2 import MME
 from magma.common.job import Job
 from magma.common.rpc_utils import grpc_async_wrapper, return_void
+from magma.common.sentry import SEND_TO_MONITORING
```

Comment:
SEND_TO_MONITORING -> SEND_TO_ERROR_MONITORING ? maybe that is more clear.

Reply:
Yes, sounds good to us. If it is okay with you we would adapt the name of the constant once there less open PRs that use it?